austrakka/components/sequence/funcs.py

```diff
@@ -298,7 +299,7 @@ def _get_seq_data(
         analysis: str,
 ):
     api_path = _get_seq_api(group_name, analysis)
-    data = call_get_api(path=api_path)
+    data = api_get(path=api_path)['data']
```

Comment:
When the App Service load balance times out, the Gateway gets a 500 error but it won't return an ApiResponse object. I think the code should be more defensive for when 'data' is null.

Reply:
I agree but this is what it was already doing

Reply:
Actually that error would bubble up and be handled. I'll make sure of that in the morning; away from the computer for the night

Reply:
Yes this error will caught. The `@_use_http_client()` decorator handles this